### ISSUE TITLE
Autosave

### ISSUE THREAD

**Issue by abdullahezzat1** [souche/vscode-mindmap#6]
I forget to save sometimes expecting changes to be saved automatically.
Please add autosave feature.
Thanks.

**Comment by JuenTingShie** [comment]
yes I think so. Auto save is important while making brain-storm, might close tabs in accident.

**Comment by Shawyeok** [comment]
At least give a prompt before close window if made changes.

**Comment by jingyibo123** [comment]
+1, maybe anyone could offer some sights on how to implement autosave functionality

**Comment by taoqf** [comment]
+1

**Comment by ff12323** [comment]
+1
